docs/run.md

```diff
@@ -128,12 +128,23 @@ https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage):
 
 4. Run LISA
 
-    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete.
+    Use above `<subscription id>` and `<private key file>` to replace
+    in commands below. They may take several minutes to complete,
+    depending on the complexity of the passed runbooks. A Marketplace
```

Comment:
the marketplace image is not a must have argument. It has the default value. If it's not. please file an issue to Lili.

Reply:
The time doesn't depends on complexity of the passed runbook here. The below command spends 3 to 5 minutes or more, the time depends on Azure's status.

Reply:
Alright, but we're not tying this patch to enter only after that, right? Whoever fixes that, fixes my docstrings as well, afterwards.

Reply:
OK, this comment is for that specific runbook, fair (about the second line).